app/bundles/CoreBundle/Helper/TemplatingHelper.php

```diff
@@ -39,21 +40,24 @@ public function __construct(Kernel $kernel)
     /**
      * Retrieve the templating service.
      *
-     * @return \Symfony\Bundle\FrameworkBundle\Templating\DelegatingEngine
+     * @return DelegatingEngine
+     *
+     * @throws \Exception
      */
     public function getTemplating()
     {
         if (defined('IN_MAUTIC_CONSOLE')) {
-            //enter the request scope in order to be use the templating.helper.assets service
-            $this->container->enterScope('request');
-            $this->container->set('request', new Request(), 'request');
+            // Use the request scope in order to be the templating.helper.assets service used
```

Comment:
This comment is out of date as the `request` scope was removed in Symfony 3:

https://symfony.com/blog/new-in-symfony-2-8-deprecating-scopes-and-introducing-shared-services

I checked the `templating.helper.assets` service and I don't see any use of Request there. Where does it fail when you remove this condition altogether?

Reply:
I can't reproduce this too without manually enabled integration in DB. GoToWebinar plugin was enabled without credentials. I checked iif I can do this in UI and it's not possible. But this fixes calls of deprecated methods.

When I enable integration in DB and remove condition it works. But this is not usual settings of app.
It had sense I believe if it is there. Request/stack is not available in container when called via CLI. I'll try to find why

Reply:
Not found. Removed.